content/en/docs/reference/glossary/aggregation-layer.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Aggregation Layer
+id: aggregation-layer
+date: 2018-10-08
+full_link: /docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
+short_description: >
+  A mechanism to extend Kubernetes by aggregating the existing API Server with additional Kubernetes-style API servers
+
+aka: 
+tags:
+- architecture
+- extension
+- operation
+---
+ A mechanism used to extend Kubernetes by aggregating the existing core API with additional Kubernetes-style APIs.
+
+<!--more--> 
+
+This mechanism must be activated on the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}}, then users can add APIService objects, which “claim” an URL path in the Kubernetes API.
```

Comment:
Replace with:
```
When you've configured the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} to [support additional APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/), you can add `APIService` objects to "claim" a URL path in the Kubernetes API.
```
So:
> When you've configured the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} to [support additional APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/), you can add `APIService` objects to "claim" a URL path in the Kubernetes API.

Reply:
Thank you for the feedback, I'll make the adjustments.
I plan to propose some more additions to the glossary, I'll make sure to follow your guidelines.